Fitness/Controllers/HomeController.swift

```diff
@@ -210,6 +227,10 @@ extension HomeController: UICollectionViewDataSource {
                 let headerView = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HomeSectionHeaderView.identifier, for: indexPath) as! HomeSectionHeaderView
                 headerView.setTitle(title: sections[indexPath.section].rawValue)
                 return headerView
+            case .pros:
+                let headerView = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: DiscoverProsSectionHeaderView.identifier, for: indexPath) as! DiscoverProsSectionHeaderView
```

Comment:
I think you can just use the `HomeSectionHeaderView` here!

Reply:
Good call, will replace it